docs/FAQ.md

```diff
@@ -210,26 +217,28 @@ information.
 
 ## I converted my config from Marlin and the X/Y axes work fine, but I just get a screeching noise when homing the Z axis
 
-Short answer: First, make sure you have verified the stepper
-configuration as described in the
-[config check document](Config_checks.md). If the problem persists,
-try reducing the max_z_velocity setting in the printer config.
-
-Long answer: In practice Marlin can typically only step at a rate of
-around 10000 steps per second. If it is requested to move at a speed
-that would require a higher step rate then Marlin will generally just
-step as fast as it can. Klipper is able to achieve much higher step
-rates, but the stepper motor may not have sufficient torque to move at
-a higher speed. So, for a Z axis with a high gearing ratio or high
-microsteps setting the actual obtainable max_z_velocity may be smaller
-than what is configured in Marlin.
+Short answer: \
```

Comment:
I don't think it is a good idea to end lines with a backslash - it isn't portable markdown and it looks confusing to users reading the text directly.

Reply:
Year but a backslash or two whitespaces are the only methods I know of making a linebreak without a heel breaking _(if this is the right word)_

Eg: <kbd>Return</kbd> vs <kbd>Shift</kbd> + <kbd>Return</kbd>

Reply:
Source:
https://spec.commonmark.org/0.31.2/#hard-line-break
https://www.markdownguide.org/basic-syntax/#line-breaks

Reply:
I have replaced " \" by "<br>" as the first source suggests it.